installation.rst

```diff
@@ -23,7 +23,7 @@ with pip pre-installed. However, if necessary you can install pip by following t
 .. warning::
 
     Python 2 will soon be deprecated by the Python maintainers. It is strongly recommended to use Python 3 with Conan, especially if need to manage non-ascii filenames or file contents.
```

Comment:
Should we include https://pythonclock.org/ as link reference? I think is a good link to know how much time Python 2 has.

Reply:
Maybe as a link in the warning box of this other section https://docs.conan.io/en/latest/installation.html#install-with-pip-recommended

Reply:
The _pythonclock_ is not related to `pip`, I would add it inside this warning or not add it at all. I'm merging this as we want to close this version, but consider adding it as a different PR.